subsys/bluetooth/shell/bredr.c

```diff
@@ -545,6 +593,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(br_cmds,
 	SHELL_CMD_ARG(oob, NULL, NULL, cmd_oob, 1, 0),
 	SHELL_CMD_ARG(pscan, NULL, "<value: on, off>", cmd_connectable, 2, 0),
 	SHELL_CMD_ARG(sdp-find, NULL, "<HFPAG>", cmd_sdp_find_record, 2, 0),
+	SHELL_CMD_ARG(pnp, NULL, "<PNP>", cmd_pnp_discover, 1, 0),
```

Comment:
There is a command `sdp-find`. we can expand the command. I don't think it is necessary to add this command.